backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -107,10 +113,22 @@ fun BpkNudger(
             size = BpkButtonSize.Default,
             type = BpkButtonType.Secondary,
             onClick = { setValue(coerced + 1) },
+            modifier = Modifier.generateNudgerTestTag(testTag, "Plus"),
         )
     }
 }
 
+@OptIn(ExperimentalComposeUiApi::class)
+private fun Modifier.generateNudgerTestTag(testTag: String?, buttonType: String): Modifier {
+    return testTag?.let {
+        semantics {
+            contentDescription = ""
+            stateDescription = ""
+            testTagsAsResourceId = true
+        }.testTag("${testTag}$buttonType")
+    } ?: this
+}
```

Comment:
using clear semantics or invisible to user won't show the test tag as resourceId in maestro, so just went with setting the contentDescription to empty, this way talkback ignore this